src/display/editor/tools.js

```diff
@@ -592,6 +613,46 @@ class AnnotationEditorUIManager {
           ["Escape", "mac+Escape"],
           AnnotationEditorUIManager.prototype.unselectAll,
         ],
+        [
+          ["ArrowLeft", "mac+ArrowLeft"],
+          AnnotationEditorUIManager.prototype.translateSelectedEditors,
+          { args: [-1, 0], checker: arrowChecker },
+        ],
+        [
+          ["ctrl+ArrowLeft", "mac+meta+ArrowLeft"],
+          AnnotationEditorUIManager.prototype.translateSelectedEditors,
+          { args: [-10, 0], checker: arrowChecker },
```

Comment:
Could we add constants for these translate-values, to make it easier to change them all at once if we ever need to in the future?
E.g. introduce something like the following somewhere above (perhaps with better names).
```javascript
const TRANSLATE_SMALL = 1; // px;
const TRANSLATE_LARGE = 10; // px;
```
Then we'd change this as follows (with similar changes below as well).
```suggestion
          ["ArrowLeft", "mac+ArrowLeft"],
          AnnotationEditorUIManager.prototype.translateSelectedEditors,
          { args: [-TRANSLATE_SMALL, 0], checker: arrowChecker },
        ],
        [
          ["ctrl+ArrowLeft", "mac+meta+ArrowLeft"],
          AnnotationEditorUIManager.prototype.translateSelectedEditors,
          { args: [-TRANSLATE_LARGE, 0], checker: arrowChecker },
```